src/index.js

```diff
@@ -157,9 +159,27 @@ function checkIsFileValidOrNot(filePath) {
   }
 }
 
+// const stringToBoolean = (stringValue) => {
+//   switch (stringValue?.toLowerCase()?.trim()) {
+//     case "true":
+//       return true;
+
+//     case "false":
+//       return false;
+
+//     default:
+//       return stringValue
+//   }
+// }
 
 function extractValues(item) {
   // Return the value of the item string as a number if possible, or the number 0 if the item string represents the number 0
+  if (item.trim() === "true") {
```

Comment:
Should convert to Boolean based on user’s value. If they wanted to convert to Boolean then have a input parameter in the function `shouldConvertToBool`. Based on that you can validate